config/software/rubygems.rb

```diff
@@ -41,7 +41,7 @@
     "2.6.8" => "40b3250f28c1d0d5cb9ff5ab2b17df6e",
   }
   known_tarballs.each do |version, md5|
-    self.version version do
+    version version do
```

Comment:
waagh. Let's change the block variable to something that doesn't shadow the method name?